### ISSUE TITLE
OSM Commands Missing

### ISSUE THREAD

**Issue by pjdufour** [locationtech/geogig#234]
The GeoGig OSM commands to update and download OSM data seem to be missing?  Were they removed, a regression, a build issue, or something else?

```
vagrant@vagrant:~$ geogig osm
osm is not a geogig command. See geogig --help.
````

**Comment by emerkle826** [comment]
The OSM and Berkeley DB (BDB) modules were removed form the project due to licensing issues in December 2016 with the 1.0-RC4 release candidate. We've moved them to a separate repository for the time being. The latest build is here: http://ares.boundlessgeo.com/geogig/dev/geogig-plugins-osm-dev-latest.zip. This plugin bundle should work with GeoGig 1.0-RC4 as well as GeoGig 1.0.
I see that we mentioned the BDB module being removed in the 1.0-RC4 release notes, but we missed the note about the OSM module being removed as well. If the licensing issues with Locationtech can be resolved in the future, we can look at bringing the OSM module back into GeoGig officially.
